build/constants.yaml

```diff
@@ -448,7 +448,7 @@
 - name: staging-runtime-constants
   constants:
     dotnetcore-staging-runtime-versions:
```

Comment:
Are we able to remove this value so the corresponding file(s) isn't generated, or is our code reliant on this file existing?

Reply:
So I wanted the runtime tests to automatically switch over to the staging image name/tags for the versions that apply, and so I added https://github.com/microsoft/Oryx/blob/main/tests/Oryx.Tests.Common/ImageTestHelper.cs#L161 into the tests. Thats the only place that I think uses the constant. I felt that leaving this empty so it still registers as a list wasn't too egregious, but if you feel strongly about it I can try and brainstorm another way to approach it.

Reply:
Got it -- I'm good with it as-is since we may re-use it in the future, I was just curious if we'd be able to remove it now since it wasn't actively being used.